instrumentation/que/lib/opentelemetry/instrumentation/que/patches/que_job.rb

```diff
@@ -37,12 +37,14 @@ def enqueue(*args, tags: nil, **arg_opts)
                 # The second option (which we are using here) is to use tags.
                 # They also are not meant for tracing information but they are
                 # much safer to use than modifying the payload.
+                tags = job_options[:tags]
+                job_class = job_options[:job_class]
                 if otel_config[:propagation_style] != :none
                   tags ||= []
                   OpenTelemetry.propagation.inject(tags, setter: TagSetter)
                 end
 
-                job = super(*args, tags: tags, **arg_opts)
+                job = super(*args, job_options: {tags: tags, job_class: job_class}, **arg_opts)
```

Comment:
Everything in `job_options` has to passed to super as well. Imagine `job_options` had a key `foo`. That would not be discarded.

Reply:
E.g.

```
job_options = job_options.merge(tags: tags)
super(*args, job_options: job_options, **arg_opts)
```
or smth similar.

Also, I don't think you need to extract `job_class` in that case either

Reply:
True. This code works. [changes[(https://github.com/open-telemetry/opentelemetry-ruby/compare/a74bf38cd592e811540fc0db05714157b54678bb..6f89f73e2da1b884c75d184be9d8268d97cffac1)